github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -198,6 +198,11 @@ periodics:
     preset-dind-enabled: "true"
     preset-docker-mirror: "true"
     preset-shared-images: "true"
+  extra_refs:
+    - org: kubevirt
+      repo: kubevirt
+      base_ref: master
+      path_alias: kubevirt
```

Comment:
You can set this repo as the work directory like this:


```
  extra_refs:
    - org: kubevirt
      repo: kubevirt
      base_ref: master
      path_alias: kubevirt
      workdir: true
```

May help you to be exactly clear where you will be inside the script.

Reply:
Sweet. I was looking for that one. Thanks